bors.toml

```diff
@@ -0,0 +1,10 @@
+# The service providing the commit statuses to GitHub.
```

Comment:
Would you like to be a guinee pig for [this](https://forum.bors.tech/t/option-to-not-include-the-pr-description-in-the-bors-merge-commit/487/2)?

Reply:
Yes! Do I just add `cut_body_after = ""` into `bors.toml`?

Reply:
I *think* so :)

Reply:
Let's try it! https://github.com/softdevteam/fm/pull/1/commits/7907c695a7b6870a58c1ab513d603fa5d40de147